test/JoeMakerV4.test.ts

```diff
@@ -175,7 +175,7 @@ describe("joeMakerV4", function () {
       await this.mimTime.transfer(this.joeMakerV4.address, await this.mimTime.balanceOf(this.dev.address))
       await this.joeMakerV4.setBridge(this.time.address, this.mim.address)
       await this.joeMakerV4.setBridge(this.mim.address, this.wavax.address)
-      await this.joeMakerV4.convert(this.mim.address, this.time.address)
+      await this.joeMakerV4.convert(this.mim.address, this.time.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.mimTime.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("213737823")
```

Comment:
Again, I think we need better assertions here; the expectation is that Bar receives a certain amount of JOE